lichess.py

```diff
@@ -40,9 +40,18 @@ def api_get(self, path):
         response.raise_for_status()
         return response.json()
 
+    # don't want to wait for max_time to accept/decline challenges if they have
+    # been withdrawn, so skip these and handle elsewhere
```

Comment:
Yes. More generally, only `5**` responses should be retried, never the `4**` responses.